api/build.gradle

```diff
@@ -6,6 +6,7 @@ plugins {
     id "net.saliman.properties" version "1.4.4"
     id "org.sonarqube" version "2.6.1"
     id "org.springframework.boot" version "1.5.2.RELEASE"
+    id "com.github.ManifestClasspath" version "0.1.0-RELEASE"
```

Comment:
This is a Windows-specific plugin, I believe we can omit it

Reply:
I've completely forgotten about it
removed